runtime/src/main/java/io/quarkiverse/logging/splunk/SplunkLogHandlerRecorder.java

```diff
@@ -25,44 +27,74 @@
 @Recorder
 public class SplunkLogHandlerRecorder {
 
-    public RuntimeValue<Optional<Handler>> initializeHandler(SplunkConfig config) {
-        if (!config.enabled) {
+    public RuntimeValue<Optional<Handler>> initializeHandler(SplunkConfig rootConfig) {
+        if (!rootConfig.config.enabled) {
             return new RuntimeValue<>(Optional.empty());
         }
+
+        Handler handler = buildHandlerFromConfig(rootConfig.config);
+        return new RuntimeValue<>(Optional.of(handler));
+    }
+
+    public RuntimeValue<Map<String, Handler>> initializeHandlers(SplunkConfig rootConfig) {
+        if (!rootConfig.config.enabled || rootConfig.namedHandlers == null || rootConfig.namedHandlers.isEmpty()) {
```

Comment:
I wonder about !rootConfig.config.enabled => should disabling the root/default handler also disable all named handler or should you disable 1 by 1 ?
Not sure what's the behavior of the core handlers (file, console, ...)